sources/web/datalab/polymer/components/details-pane/details-pane.html

```diff
@@ -0,0 +1,87 @@
+<!--
+Copyright 2017 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License
+is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+or implied. See the License for the specific language governing permissions and limitations under
+the License.
+-->
+
+<link rel="import" href="../../components/shared-styles/shared-styles.html">
+
+<link rel="import" href="../../bower_components/iron-icons/iron-icons.html">
+<link rel="import" href="../../bower_components/paper-button/paper-button.html">
+
+<dom-module id="details-pane">
+  <template>
+    <style include="datalab-shared-styles">
+      .placeholder {
+        padding: 15px;
+        text-align: center;
+      }
+      #container {
+        padding: 0px 20px;
+        height: 100%;
+        overflow-y: auto;
+      }
+      #details-pane-title {
+        display: flex;
+      }
+      .content {
+        padding: 15px 3px;
+        margin: 0px;
+      }
+      .icon {
+        min-height: 30px;
+        min-width: 30px;
+      }
+      #previewHtml {
+        background-color: #fff;
+        padding: 0px 10px;
+        margin: 10px 0px;
+        overflow-y: auto;
+        overflow-x: hidden;
+        box-shadow: 5px 5px 10px -3px rgba(0, 0, 0, 0.1);
+      }
+    </style>
+    <div class="placeholder" hidden$="{{file}}">
+      Select an item to view its details.
+    </div>
+    <div id="container" hidden$="{{!file.name}}">
+      <div id="details-pane-title">
+        <iron-icon class="content icon" icon="{{_icon}}"></iron-icon>
+        <h1 class="content">{{file.name}}</h1>
+      </div>
+      <hr/>
+      <div class="content">
+        <table>
+          <tr>
+            <td>
+              <strong>Created:</strong>
```

Comment:
Rather than using &lt;strong&gt; here, I think it would be better to do that styling in css. You could either put a class on this &lt;td&gt;, or use &lt;th&gt;.